spec/kitchen/provisioner/chef_base_spec.rb

```diff
@@ -284,6 +293,25 @@ def calculate_path(path, _opts = {})
         cmd
       end
 
+      it "will use chef_omnibus_cache to provide a cache directory" do
```

Comment:
So you confirmed that it won't clobber the -d. Can you add a test case where we append -d to install_options that already include other values?